apps/nextjs/src/app/[locale]/manage/users/groups/[id]/members/page.tsx

```diff
@@ -0,0 +1,99 @@
+import Link from "next/link";
+import {
+  Anchor,
+  Center,
+  Group,
+  Stack,
+  Table,
+  TableTbody,
+  TableTd,
+  TableTr,
+  Text,
+  Title,
+} from "@mantine/core";
+
+import type { RouterOutputs } from "@homarr/api";
+import { api } from "@homarr/api/server";
+import { getScopedI18n } from "@homarr/translation/server";
+import { SearchInput, UserAvatar } from "@homarr/ui";
+
+import { AddGroupMember } from "./_add-group-member";
+import { RemoveGroupMember } from "./_remove-group-member";
+
+interface GroupsDetailPageProps {
+  params: {
+    id: string;
+  };
+  searchParams: {
+    search: string | undefined;
+  };
+}
+
+export default async function GroupsDetailPage({
+  params,
+  searchParams,
+}: GroupsDetailPageProps) {
+  const tMembers = await getScopedI18n("management.page.group.setting.members");
+  const group = await api.group.getById({ id: params.id });
+
+  const filteredMembers = searchParams.search
+    ? group.members.filter((member) =>
+        member.name
+          ?.toLowerCase()
+          .includes(searchParams.search!.trim().toLowerCase()),
+      )
+    : group.members;
+
+  return (
+    <Stack>
+      <Title>{tMembers("title")}</Title>
+      <Group justify="space-between">
+        <SearchInput
+          placeholder={`${tMembers("search")}...`}
```

Comment:
Maybe include this in the translation?

Reply:
This will be part of https://github.com/homarr-labs/homarr/issues/366